src/radtest/radtest.c

```diff
@@ -55,6 +56,9 @@ void rtst_main (void){
 		if ((RtstTickCnt % RTST_MEMTST_TICKS) == 0) {
 			// Memory tests TODO
 		}
+		if ((RtstTickCnt % RTST_SENSOR_TICKS) == 0) {
+			SenReadAllValues();
```

Comment:
![image](https://user-images.githubusercontent.com/66519643/183971420-759a07cd-7e74-4095-982a-e38c8dc34078.png)


Please check if this was indended for ISSUE-33